uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsMockMvcTests.java

```diff
@@ -1068,6 +1069,23 @@ void testUpdateUser_No_Username_Returns_400() throws Exception {
         updateUser(scimReadWriteToken, HttpStatus.BAD_REQUEST.value());
     }
 
+    @Test
+    void testUpdateUser_ChangingOriginReturns400() throws Exception {
+        final ScimUser scimUser = setUpScimUser(IdentityZone.getUaa());
+
+        // change origin
+        final String initialOrigin = scimUser.getOrigin();
+        scimUser.setOrigin(UUID.randomUUID().toString());
+
+        scimUser.setName(new ScimUser.Name("Some New", "Name"));
+
+        updateUser(scimReadWriteToken, HttpStatus.BAD_REQUEST.value(), scimUser);
+
+        // should work again when origin was not changed
```

Comment:
This last part doesn't seem necessary for the test, and should be covered by a different test.

Reply:
fixed with 0dec2a7